client/asset/btc/spv.go

```diff
@@ -1244,6 +1241,55 @@ func (w *spvWallet) startWallet() error {
 	return nil
 }
 
+// deleteWalletData will delete all files except peers.json from the
+// wallet's directory.
+func (w *spvWallet) deleteWalletData() error {
+	contents, err := os.ReadDir(w.netDir)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range contents {
+		if f.Name() == "peers.json" {
+			continue
+		}
+		path := filepath.Join(w.netDir, f.Name())
+		if f.IsDir() {
+			_ = os.RemoveAll(path)
+		} else {
+			_ = os.Remove(path)
+		}
```

Comment:
I THINK `RemoveAll` is find for both files and directories. Is it not?

Reply:
Yes you're right.